wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Follower Totals

Fixes #10513 

This adds the `Follower Totals` card to the Insights table.

NOTE: We don't currently have a way to obtain the Social total. So for now, a fake value is displayed.

Hack tip: To show/hide any of the three follower types, in `StatsInsightsStore`, add a `return ""` to any of these three methods:
- `getTotalDotComFollowers`
- `getTotalEmailFollowers`
- `getTotalPublicizeFollowers`

To test:

---
- On a site with WP, Email, and Social followers, go to Stats > Insights.
  - Tip: en.blog.wordpress.com has lots.
- Verify all three rows appear:

<img width="465" alt="followers_all" src="https://user-images.githubusercontent.com/1816888/49117151-08e66f80-f25d-11e8-81f1-95af5e497bf2.png">

---
- On a site that has some of the three follower types, go to Stats > Insights.
- Verify only those that have values appear:

<img width="462" alt="followers_some" src="https://user-images.githubusercontent.com/1816888/49117169-14d23180-f25d-11e8-9dfb-47bb23e61253.png">

---
- On a site that has no followers, go to Stats > Insights.
- Verify `No data yet` appears:

<img width="465" alt="followers_none" src="https://user-images.githubusercontent.com/1816888/49117190-21568a00-f25d-11e8-8458-559aad5eed1c.png">
